docs/testbed/README.testbed.VsSetup.md

```diff
@@ -4,29 +4,36 @@ This document describes the steps to setup a virtual switch based testbed, deplo
 ## Prepare testbed host
 First, we need to prepare the host where we will be configuring the virtual testbed and running the tests.
 
-1. Install Ubuntu 20.04 AMD64 on your host or VM
+1. Install Ubuntu AMD64 on your host or VM
     - To setup a T0 topology, the server needs to have at least 10GB of memory free
     - If the testbed host is a VM, then it must support nested virtualization
         - [Instructions for Hyper-V based VMs](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization)
-2. Ensure that python and pip are installed
-```
-apt install python3 python3-pip
-```
-
-3. If the server was upgraded from Ubuntu 18.04, check the default python version using command `python --version`. If the default python version is still 2.x, replace it with python3 using symbolic link:
-```
-ln -sf /usr/bin/python3 /usr/bin/python
-```
-
-4. Run the host setup script to install required packages and initialize the management bridge network
+2. Prepare your environment based on different Ubuntu version, make sure that python and pip are installed
+   1. Option : If your host is **Ubuntu 20.04**
+        
+        ```
+        apt install python3 python3-pip
+        ```
+        If the server was upgraded from Ubuntu 18.04, check the default python version using command `python --version`. If the default python version is still 2.x, replace it with python3 using symbolic link:
+        ```
+        ln -sf /usr/bin/python3 /usr/bin/python
+        ```
+   2. Option : If your host is **Ubuntu 18.04**
+        ```
+        apt install python python-pip
+        # v0.3.12b Jinja2 is required, lower version may cause uncompatible issue
+        pip install j2cli==0.3.12b0
```

Comment:
I think prepending `sudo` to the command is required.

Reply:
added sudo for apt instll python python-pip
and pip install j2cli==0.3.12b0 doesn't need a sudo